src/Pages/ara/Login.js/Footer.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./Footer.scss";
+import { Link } from "react-router-dom";
+import { withRouter } from "react-router-dom";
```

Comment:
- import 순서 지켜주세요.
- react router dom에서 Link, withRouter를 각각 가져오고 있는데 한번에 가져올 수 있습니다.
```suggestion
import React from "react";
import { Link, withRouter } from "react-router-dom";
import "./Footer.scss";
```